app/models/account.rb

```diff
@@ -95,6 +96,28 @@ def reset!
     RedisEndpoint.reset!
   end
 
+  # Get all administrator emails associated with this account
+  def admin_emails
+    # Must ensure we are switched to proper tenant database
+    Apartment::Tenant.switch(tenant) do
+      # return list of email addresses of all users with admin role on this site
+      User.with_role(:admin, Site.instance).map(&:email)
```

Comment:
There may be some small amount of optimization available here by using `.pluck(:email)` instead of `.map(&:email)`, since that moves the operation into the DB query.